app/helpers/request_helper.rb

```diff
@@ -0,0 +1,58 @@
+module RequestHelper
+  def recent_item_issue_held(holding)
+    most_recent = holding["holdingsStatements"].last
+
+    if most_recent.present?
+      if most_recent["statement"].present?
+        most_recent["statement"]
+      elsif most_recent["note"].present?
+        most_recent["enumeration"]
+      end
+    end
+  end
```

Comment:
It is also possible for both a statement and a note to be present, as a note applying to the statement.

Reply:
Also munged statements and notes together using the helper method.